src/Kestrel.Core/Internal/Http2/HPack/HeaderField.cs

```diff
@@ -12,6 +12,9 @@ public HeaderField(string name, string value)
         }
 
         public string Name { get; }
+
         public string Value { get; }
+
+        public int Length => Name.Length + Value.Length + 32;
```

Comment:
32?

Reply:
http://httpwg.org/specs/rfc7541.html#rfc.section.4.1

Reply:
Can you generally put more comments in the code that are related to the spec.